### PR TITLE
Address EIP text suggestion

### PR DESCRIPTION
**Pull request by ThunderDeliverer**
The EIP has been updated to reflect the changes we have made and to address the necessary changes.

### CODE REVIEW

**Comment by ThunderDeliverer** [comment]
@steven2308 I'll leave the `Rationale` update to you and then pick up the rest in the morning (if you have any suggestions, I'll address them then as well).

If you have a suggestion on the JSON format of the metadata, you can leave it here as well.

**Review by steven2308 — APPROVED** [review]
Let's go!

File: README.md [hunk 1]
```diff
@@ -478,21 +478,18 @@ The MultiResource token standard has been made compatible with [EIP-721](./eip-7
 
 ## Test Cases
 
-The RMRK MultiResource lego block implementation includes test cases written using Hardhat.
+Tests are included [`here`](../assets/eip-5773/test/multiresource.ts).
```
Comment (ThunderDeliverer) [inline README.md:481]:
```suggestion
Tests are included in [`multiresource.ts`](../assets/eip-5773/test/multiresource.ts).
```

File: README.md [hunk 2]
```diff
@@ -478,21 +478,18 @@ The MultiResource token standard has been made compatible with [EIP-721](./eip-7
...
+Run in terminal
```
Comment (ThunderDeliverer) [inline README.md:482]:
```suggestion

To run them in terminal, you san use the following commands:
```

**Pull request merged**